assets/css/main.css

```
@@ -91,6 +91,33 @@ a:hover {
 
 /* FAQ
 ================================ */
+.accordion-button::after {
+  background-image: url(../img/plus-drop-down.svg);
+  }
+
+.accordion-button:not(.collapsed)::after {
+  background-image: url(../img/minus-drop-down.svg);
+}
+
+.accordion-button:focus {
+  border-color: rgba(0,0,0,.125);
+  box-shadow: none;
+}
+
+.accordion-button:not(.collapsed) {
+  color: black;
+  background-color: white;
+  border-bottom: white;
```

Rather than coding `white` for our current background color, it's safer to set these values to be `transparent`. Also, the design team seemed to want an underline below the FAQ titles when they're expanded.

```suggestion
  background-color: transparent;
```

---

I'm having trouble removing the line between an answer and the question below it when

---

I would say you're fine keeping the border on bottom. I might play around with some CSS selectors later to see if there's some way to get it exactly like the figma.